.gitmodules

```diff
@@ -1,15 +1,21 @@
 [submodule "sonic-swss-common"]
 	path = src/sonic-swss-common
-	url = https://github.com/Azure/sonic-swss-common
+	url = https://github.com/barefootnetworks/sonic-swss-common
 [submodule "sonic-linux-kernel"]
 	path = src/sonic-linux-kernel
 	url = https://github.com/Azure/sonic-linux-kernel
 [submodule "sonic-sairedis"]
 	path = src/sonic-sairedis
-	url = https://github.com/Azure/sonic-sairedis
+	url = https://github.com/barefootnetworks/sonic-sairedis
```

Comment:
can you open PR against sonic-sairedis repo on 201712 branch, we'll take the patch at that repo, then update the reference here.